packages/store/src/types.js

```diff
@@ -339,14 +339,14 @@
  * key order equivalence class is always at least as precise as the
  * rank order equivalence class. IOW, if `compareKeys(X,Y) === 0` then
  * `compareRank(X,Y) === 0`. But not vice versa. For example, two different
- * remotables are the same rank but incommensurate as keys.
+ * remotables are the same rank but incomparable as keys.
  *
  * A further invariant is if `compareKeys(X,Y) < 0` then
  * `compareRank(X,Y) <= 0`, i.e., if X is smaller than Y in key order, then X
  * must be at least as early as Y in rank order. But not vice versa.
  * X can be earlier than Y in rank order and still be incommensurate with Y in
- * key order. For example, the record `{b: 3, a: 5}` is earlier than but
- * incommensurate with the record `{b: 5, a: 3}`.
+ * key order. For example, the record `{b: 3, a: 5}` is earlier than
+ * the record `{b: 5, a: 3}` in rank order but they are incomparable as keys.
```

Comment:
Is there an example of values that have the _same_ rank but are incomparable as keys, e.g. `compareRank(X,Y) === 0 && Number.isNaN(compareKeys(X,Y))`? That would be good to include here as well.

Reply:
Two remotables
```js
const X = Far('X', {});
const Y = Far('Y', {});
compareKeys(X,Y); // should be NaN
compareRank(X,Y); // should be 0
```